contrib/marisa-trie/template.py

```diff
@@ -0,0 +1,33 @@
+pkgname = "marisa-trie"
+pkgver = "0.2.6"
+pkgrel = 0
+build_style = "gnu_configure"
+configure_args = ["--enable-static"]
+make_dir = "."
+hostmakedepends = [
+    "autoconf",
+    "automake",
+    "clang",
```

Comment:
```suggestion
    "automake",
```

Reply:
well, just copy the version from https://github.com/chimera-linux/cports/pull/1957 for this, though i think i will add this myself for something else first